lib/Github/Api/Apps.php

```diff
@@ -5,28 +5,14 @@
 use Github\Api\AcceptHeaderTrait;
 
 /**
- * @link   https://developer.github.com/v3/integrations/
+ * @link   https://developer.github.com/v3/apps/
  * @author Nils Adermann <naderman@naderman.de>
  */
-class Integrations extends AbstractApi
+class Apps extends AbstractApi
```

Comment:
As I also modified the Client to point to the new class, there's no reason to keep using this...